lib/server.js

```diff
@@ -891,13 +891,15 @@ Server.prototype._run = function _run(req, res, route, chain, cb) {
                 return (next());
             }
 
+            handlerName = (chain[i].handlerName ||
```

Comment:
@LeMisterV Presumably ```handlerName``` is a property on the function that you'd set when you initialize the route? If so we'll need to update docs to reflect this before we merge.